common/acpi.c

```diff
@@ -39,6 +39,11 @@ unsigned max_acpi_tables;
 
 static unsigned nr_cpus;
 
+unsigned acpi_get_nr_cpus(void) { return nr_cpus; }
+
+#define ACPI_NR_TABLES(ptr)                                                              \
```

Comment:
How about adding a comment what the logic behind this calculation is? I'm not getting it on first read.